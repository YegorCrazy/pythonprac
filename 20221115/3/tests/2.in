try:
	a = Alpha(aaa = 3)
except AttributeError:
	print('No aaa')

try:
        a = AlphaQ(bbb = 3)
except AttributeError:
        print('No bbb')

a = Alpha(a = 3)
b = AlphaQ(b = 3)

try:
	print(a.b)
except AttributeError:
	print('No a.b')

try:
        print(b.a)
except AttributeError:
        print('No b.a')
