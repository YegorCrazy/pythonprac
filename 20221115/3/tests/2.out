No aaa
No bbb
No a.b
No b.a
